app/views/layouts/_head.html.erb

```diff
@@ -41,7 +41,7 @@
 
   // Interaction tracking
   $(document).ready(function(e) {
-    TrackLinks( $( '.region a' ) );
+    TrackLinks( $( 'a' ) );
```

Comment:
This will track all links with Events, internal and external... will that result in double counting stats? (one for the normal GA code and one for the Event?). I think it's probably okay, I just want to make sure we aren't going to mislead ourselves in terms of usage later on.